src/models/Lessons.js

```diff
@@ -111,70 +111,95 @@ export async function generateLessons(sequelize, models) {
   return Lessons;
 }
 
-export function getAll() {
-  return Lessons.findAll()
-    .then(lessons => lessons)
-    .catch(() => 'fail');
+export async function getAll() {
+  try {
+    return await Lessons.findAll();
+  } catch (err) {
+    throw new Error(err);
+  }
```

Comment:
What it the purpose of such `catch`? What is difference in behavior with following code
```js
export function getAll() {
  return Lessons.findAll();
}
```